src/main/java/org/spongepowered/common/event/SpongeCommonEventFactory.java

```diff
@@ -137,6 +137,9 @@
 
 public final class SpongeCommonEventFactory {
 
+    private static final double MOVEMENT_GRID_POINTS_PER_BLOCK = 8.0d;
```

Comment:
I meant to put this in my review, but my question was, why do 8? The delta change before the idea behind `api8/move-event-grid` had a distance of 1/16, so why change it to 1/8?

Reply:
Why did I do 16 in move-event-grid? It's somewhat arbitary, trying to strike a balance between firing significant events, but not firing too many.

I may make this configurable for performance reasons.

Reply:
Fair enough!